rudder-core/src/main/scala/com/normation/rudder/services/reports/NodeChangesService.scala

```diff
@@ -342,13 +325,10 @@ object NodeChanges {
     d1 isBefore d2
   }
 
-
-
   val data = changesByPeriod(changes).map(a => (a._1,a._2.size))
     JsObj(
         ("x" -> JsArray(data.map(a => Str(a._1))))
-      , ("y" -> JsArray(data.map(a => Num(a._2))))
+      , ("y" -> JsArray(data.map(a => Num(Random.nextInt(10000)))))
```

Comment:
It was to display good chubby data in my graph (my dev env has 6 changes in 23 rules, most of theml disabled ..)